Source/WebCore/dom/AbortSignal.cpp

```diff
@@ -79,7 +79,9 @@ Ref<AbortSignal> AbortSignal::any(ScriptExecutionContext& context, const Vector<
 {
     Ref resultSignal = AbortSignal::create(&context);
 
-    auto abortedSignalIndex = signals.findIf([](auto& signal) { return signal->aborted(); });
+    auto abortedSignalIndex = signals.findIf([](auto& signal) {
```

Comment:
Note for reviewer: Did `check-webkit-style` change config? Running `Tools/Scripts/check-webkit-style Source/WebCore/dom/AbortSignal*` failed at this line, so fixed. Arguably unrelated change to this PR.